cmonge/trainers/ot_trainer.py

```diff
@@ -199,6 +199,7 @@ def setup(
         fitting_loss: Dict[str, Any],
         regularizer: Dict[str, Any],
         optim: Dict[str, Any],
+        checkpointing_path: None,  # For compatability with base class
```

Comment:
`None` is an expression but you use it as type hint here, I reckon you actually mean this?

```suggestion
        checkpointing_path: Optional[str] = None,  # For compatibility with base class
```